src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -20,6 +20,7 @@ export default class ReviewCollapsibleChapter extends React.Component {
   constructor() {
     super();
     this.handleEdit = this.handleEdit.bind(this);
+    this.state = { editing: false };
```

Comment:
Tracking whether or not we are editing as a state variable will allow us to determine whether or not to prepend the chapter header with "Edit"